main.go

```diff
@@ -90,14 +90,14 @@ func (p *Package) findLicense() (string, error) {
 		// Look for a LICENSE* file in the package directory (or parents) instead
 		licenseFile, err := findLicenseFileUp(p.Dir)
 		if err != nil {
-			return "", errors.Wrapf(err, "finding license file for %s", p.ImportPath)
+			return "Unknown", errors.Wrapf(err, "finding license file for %s", p.ImportPath)
```

Comment:
this kind of human-readable default is better done higher up the call stack IMHO.